src/app/events/[slug]/services/lumaService.ts

```diff
@@ -0,0 +1,40 @@
+import { z } from 'zod'
+
+import { type LumaEventEntry, LumaResponseSchema } from '../schemas/LumaSchema'
+
+export async function getLumaEvents(): Promise<Array<LumaEventEntry>> {
+  const apiKey = process.env.NEXT_LUMA_FILECOIN_API_KEY
+  if (!apiKey) throw new Error('Luma API key is not configured')
+
+  try {
```

Comment:
The `try catch` might be unnecessary here, depending on how we want to handle the error state.